DP1.10058.001/Image.legend1.txt

```diff
@@ -1 +1 @@
-The square, multi-scale plot used to record plant species composition and cover. The plot includes nested subplots at specific locations within the plot. The 100m^2 subplot naming (e.g., 31, 32, 40, 41) corresponds to the point identification (see for point identification logic and description) in the southwest corner of the subplot. The 100m^2 subplot corners are numbered counter clockwise starting in the southwest corner. The 100m^2 subplot identifier, the subplot corner, and the scale of observation name the 1 and 10m^2 subplots. Subplot 32.2.1, for example, is the 1m^2 subplot in corner 2 of 100m^2 subplot 32. 
+The square, multi-scale plot used to record plant species composition and cover. The plot includes nested subplots at specific locations within the plot. The 100m^2 subplot naming (e.g., 31_100, 32_100, 40_100, 41_100) corresponds to the identity of the point in the southwest corner of the subplot. Smaller subplots (i.e., 1m^2 and 10m^2) are named according to the 100m^2 subplots within which they are nested, the subplot scale, and the corner of the 100m^2 subplot; corners within the 100m^2 subplot are numbered counterclockwise starting in the southwest corner. Subplot 32_1_2, for example, is the 1m^2 subplot in corner 2 of 100m^2 subplot 32_100. Prior to 2019, there were additional subplots near the center of the plot: 31_1_4, 31_10_4, 41_1_1, and 41_10_1 that were discontinued.
```

Comment:
Is there a document we can link to here? Something like "For full details on subplot naming conventions, see:"

Reply:
added: "see NEON.DOC.005108 NEON Terrestrial Observation System: Spatial and Temporal Sampling Strategy, and NEON.DOC.014042 TOS Protocol and Procedure: DIV - Plant Diversity Sampling."